### ISSUE TITLE
Migrate rivet.iu.edu to Sitehost

### ISSUE THREAD

**Issue by levimcg** [indiana-university/rivet-source#204]
With the retirement of Webserve approaching, we need to prioritize the migration of rivet.iu.edu to Sitehost.

The Sitehost service has been enabled for the main Rivet group account. Once we have updated our CI/publishing processes and can confirm that they are working correctly, we should request that the virtual host be switched over to Sitehost. See this KB doc for more info on migrating:

https://kb.iu.edu/d/axoh

@illusivesunrae @zttodd you both should have access to the group account.

**Comment by illusivesunrae** [comment]
I cloned the Bamboo plans for `rivet.iu.edu` (`rivet.sitehost.iu.edu`) and `rivet.webtest.iu.edu` (`rivet.sitehost-test.iu.edu`). I updated the plans to reflect the move to `sitehost`, and confirmed that they worked. 

@levimcg has requested that the Virtual Host (`rivet.iu.edu`) be moved to point to `rivet.sitehost.iu.edu`.

Once DNS has updated the Virtual Host, I will remove then superfluous Bamboo plans, rename the `rivet.sitehost.iu.edu` plan to `rivet.iu.edu`. Then we should be done with the migration of the current documentation site to sitehost.
